Groups.md

```diff
@@ -49,7 +45,7 @@ UIC Homebrew Computer Club (Abbreviation: UICHCC) is an interest-driven organiza
 
 **TBD: ** To be decided
 
-### Cyber Security & Self-Protection 101
+### Cyber Security & Self-Protection Course
```

Comment:
I said 101 was okay (although I have no idea what is it)

Reply:
[101 (number): In education](https://en.wikipedia.org/wiki/101_(number)#In_education)